normandy/recipes/filters.py

```diff
@@ -547,6 +577,21 @@ def capabilities(self):
         return {"jexl.transform.date"}
 
 
+class WindowsBuildNumberFilter(BaseComparisonFilter):
+    type = "windows_build_number"
+
+    @property
+    def left_of_operator(self):
+        return "normandy.os.windowsBuildNumber"
+
+    @property
+    def capabilities(self):
+        return set()
+
+    def to_jexl(self):
+        return f"(normandy.os.isWindows && {super().to_jexl()})"
+
```

Comment:
@rehandalal I put parentheses around this because that seems to be how it's often represented in the filter expressions... but I am guessing they don't actually matter in this case because it's an `&&` operation. Is that correct?

Reply:
yeah given that it's an `&&` it should not matter either way

Reply:
@rehandalal So in the case where the parentheses don't matter, even if it often appears that way in the filter expressions of past recipes, would you prefer I left the parentheses out or use them just for a bit of extra clarity?

Reply:
It doesn't matter today, since everything is `&&`, but it might matter in the future if we add an OR filter. Lets keep the parens.

Reply:
OK, well I inserted Rehan's example above, and kept the parens, so let me know what y'all think of it now.